src/api/admin_user/handlers/delete_admin_user_handler.rs

```diff
@@ -9,12 +9,16 @@ use axum::{
 };
 
 pub async fn delete_admin_user_handler(
-    _session: AvoRedSession,
+    mut _session: AvoRedSession,
```

Comment:
remove the underscore as it should only be used when we are not using the variable.